demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -143,7 +147,29 @@ def update_tests(self) -> None:
                             test_playbook_data = get_yaml(tpb_file_path)
                             test_playbook_id = get_entity_id_by_entity_type(test_playbook_data,
                                                                             content_entity='')
-                            test_playbook_ids.append(test_playbook_id)
+                            if not scripts and not commands:  # Better safe than sorry
+                                test_playbook_ids.append(test_playbook_id)
```

Comment:
What cases could be that TPB will not have scripts or commands and we will want to add him? afraid of cases where the function could have missed?

Reply:
Yes, like indirect subplaybooks. I think if the script missed something then the developer should see that all test playbooks are running and then fix accordingly. I think this is better than not adding the test playbooks & short passing pipelines making the developer think all is good.